lib/board.ex

```diff
@@ -4,25 +4,21 @@ defmodule Board do
     0..last_board_space |> Enum.to_list
   end
 
-  def is_available?(board, space) do
-    Enum.at(board, space) != "X" and Enum.at(board, space) != "O"
+  def is_available?(game, space) do
+    Enum.at(game.board, space) != game.player_1.mark and Enum.at(game.board, space) != game.player_2.mark
```

Comment:
I think `player_1` and `player_2` can sometimes be nil, in an empty `Game`.  If that's right, then this should maybe have a `cond` block to handle it (and a test to prove it).